ci/game/project_setup.py

```diff
@@ -0,0 +1,98 @@
+import json
+import os
+import subprocess
+import platform
+from contextlib import contextmanager
+
+
+# These Conan repos must exist in the server, with these names (local repos)
+DEVELOP = "develop"
+PACKAGES = "packages"
+PRODUCTS = "products"
+# TODO: This must be configured by users
+SERVER_URL = os.environ.get("ARTIFACTORY_URL", "http://localhost:8081/artifactory/api/conan")
+USER = os.environ.get("ARTIFACTORY_USER", "admin")
+PASSWORD = os.environ.get("ARTIFACTORY_PASSWORD", "password")
+
+
+def run(cmd, error=False, env_script=None, file_stdout=None):
+    if env_script is not None:
+        env = "{}.bat".format(env_script) if platform.system() == "Windows" else ". {}.sh".format(env_script)
+        cmd = "{} && {}".format(env, cmd)
+    # Used by tools/scm check_repo only (see if repo ok with status)
+    print("Running: {}".format(cmd))
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True, text=True)
+    out, err = process.communicate()
+    ret = process.returncode
+
+    if file_stdout:
+        open(file_stdout, "w").write(out)
+
+    output = err + out
+    if ret != 0 and not error:
+        raise Exception("Failed cmd: {}\n{}".format(cmd, output))
+    if ret == 0 and error:
+        raise Exception("Cmd succeded (failure expected): {}\n{}".format(cmd, output))
+    return output
+
+
+def clean():
+    run('conan remove "*" -c')  # Make sure no packages from last run
+    run('conan remote remove "*"')
+
+
+def add_repo(name):
+    run(f"conan remote add {name} {SERVER_URL}/{name}")
+    run(f"conan remote login {name} {USER} -p {PASSWORD}")
```

Comment:
```suggestion
    run(f"conan remote login {name} {USER} -p '{PASSWORD}'")
```

otherwise a password that contains an ampersand `&` will fail here. I just hit this running through the tutorially locally with an autogenerated password containing an ampersand

Reply:
Problem with single quotes is that it is not understood the same in some terminals, generally double quotes work better. 

This has changed a lot in the last review of the tutorial, the ``add_repo`` function doesn't exist anymore. I am going to change it for double quotes. Thanks for pointing this out!